rules/network/initial_access_unsecure_elasticsearch_node.toml

```
@@ -38,9 +38,10 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-(event.dataset: network_traffic.http OR (event.category: network_traffic AND network.protocol: http)) AND
-    status:OK AND destination.port:9200 AND network.direction:inbound AND NOT http.response.headers.content-type:"image/x-icon" AND NOT
-    _exists_:http.request.headers.authorization
+(event.dataset:network_traffic.http or (event.category:network_traffic and network.protocol:http)) and
+    http.response.status_phrase:ok and destination.port:9200 and network.direction:inbound and
+    not http.response.mime_type:"image/x-icon" and
+    not http.request.headers.authorization:*
```

pending upstream changes

```suggestion
    not http.request.Headers.authorization:*
```

---

I'm reconsidering this remapping. Please wait for the integrations PR so you have a concrete target to work against.

---

based on https://github.com/elastic/integrations/pull/8221

```suggestion
    not network_traffic.http.request.headers.authorization:*
```